src/lumigo_tracer/w3c_context.py

```diff
@@ -0,0 +1,59 @@
+import re
+import random
+from typing import Dict, Optional
+
+# parts of this files were copied from:
```

Comment:
Shouldn't it be in the "libs" directory?

Reply:
This is a misleading comment, only a small part was copied. I'll rephrase it!